deploy/olm-catalog/windows-machine-config-operator/manifests/windows-machine-config-operator.clusterserviceversion.yaml

```diff
@@ -188,17 +184,16 @@ spec:
   installModes:
   - supported: true
     type: OwnNamespace
-  - supported: false
+  - supported: true
     type: SingleNamespace
   - supported: false
     type: MultiNamespace
-  - supported: false
+  - supported: true
     type: AllNamespaces
   keywords:
-  - windows
+  - ""
   maintainers:
   - {}
   maturity: alpha
-  provider:
-    name: Red Hat
```

Comment:
Not sure why `keywords` and `provider` are empty here

Reply:
This should not be changed unless there is a reason